code-challenges.test.js

```diff
@@ -10,45 +10,85 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+var colors1 = ["purple", "blue", "green", "yellow", "pink"];
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+var colors2 = [
+  "chartreuse",
+  "indigo",
+  "periwinkle",
+  "ochre",
+  "aquamarine",
+  "saffron",
+];
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
-
+describe("colorShuffler", () => {
+  const expected1 = ["yellow", "blue", "pink", "green"];
+  const expected2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"];
+  it("returns an array with the first index removed with the rest of the values in the array shuffled", () => {
+    expect(colors1).toEqual(expect.arrayContaining(expected1));
+    expect(colors2).toEqual(expect.arrayContaining(expected2));
+  });
+});
+
+const colorShuffler = (array) => {
+  array.shift();
+  let newArray = array.sort((a, b) => a.sort - b.sort);
+  array.map(({ value }) => value);
+  console.log(newArray);
+};
 
 // b) Create the function that makes the test pass.
 
-
-
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
+var nums1 = [3, 56, 90, -8, 0, 23, 6];
 // Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
+var nums2 = [109, 5, 9, 67, 8, 24];
 // Expected output: [5, 109]
 
-
+describe("minMaxNum", () => {
+  it("returns an array with the minimum and maximum number", () => {
+    expect(minMaxNum(nums1)).toEqual([-8, 90]);
+    expect(minMaxNum(nums2)).toEqual([5, 109]);
+  });
+});
 
 // b) Create the function that makes the test pass.
 
-
+const minMaxNum = (array) => {
+  let newArray = []; // created a new empty array to put the minimum and maximum number
+  newArray[0] = Math.min(...array); //using the spread operator to find the smallest number in the arrat and assigning it to the first index
+  newArray[1] = Math.max(...array); //using the spread operator to find the largest number in the arrat and assigning it to the first index
+  return newArray;
+};
```

Comment:
Nice job. I like this logic. I think you could make it even more concise by putting your min/max in an array. `return [Math.min(...array), Math.max(...array)]`